<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDownloadDialog</class>
 <widget class="QDialog" name="VideoDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>URLs:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edtURLs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Download subtitles:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtSubtitles">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Comma-separated list of languages (for full list search &quot;ISO 639-1&quot;):&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;en&lt;/b&gt; (english)&lt;/li&gt;
&lt;li&gt;&lt;b&gt;ru&lt;/b&gt; (russian)&lt;/li&gt;
&lt;li&gt;&lt;b&gt;es&lt;/b&gt; (spanish)&lt;/li&gt;
&lt;li&gt;&lt;b&gt;it&lt;/b&gt; (italian)&lt;/li&gt;
&lt;li&gt;&lt;b&gt;zh&lt;/b&gt; (chinese)&lt;/li&gt;
&lt;li&gt;etc...&lt;/li&gt;
&lt;/ul&gt;
The asterisk symbol (*) sets the pattern e. g. &quot;&lt;b&gt;en*&lt;/b&gt;&quot; = &quot;&lt;b&gt;en-US, en-GB&lt;/b&gt;&quot;
</string>
       </property>
       <property name="text">
        <string>en*,ru*</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Prefer resolution: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="edtResolution">
       <property name="toolTip">
        <string>Download the best video available with the largest height but no better than specified resolution, or the best video with the smallest resolution if there is no video under than specified</string>
       </property>
       <property name="suffix">
        <string>p</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>720</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Report:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edtReport">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDownload">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
